tests/Query/Filter/DueDateField.test.ts

```diff
@@ -47,6 +47,51 @@ describe('due date', () => {
         testTaskFilterForTaskWithDueDate(filter, '2022-00-01', true); // month 0 not valid
         testTaskFilterForTaskWithDueDate(filter, '2022-13-01', true); // month 13 not valid
     });
+
+    it.each([
+        // Week
+        ['due in this week', '2022-01-09 (Sunday 9th January 2022)', false],
+        ['due in this week', '2022-01-10 (Monday 10th January 2022)', true],
+        ['due in this week', '2022-01-16 (Sunday 16th January 2022)', true],
+        ['due in this week', '2022-01-17 (Monday 16th January 2022)', false],
```

Comment:
> How did you manage to resolve the problem of the Sunday/Monday week start date when running the tests in GitHub Actions?

By using 'isoWeek' instead of 'week' in the momentjs call

Reply:
Indeed. And for the documentation, users will need it to be expressed in a way that does not require any programming knowledge at all. They shouldn’t need to know what an isoWeek is, for example.

Reply:
Of course. Sorry, a bit unclear for me. Do you mean using isoWeek is wrong since it needs to be explained in the docs so `week` should be used with locale setting with jest? Or keep things as they are?

Reply:
In the query user shall write `this week`, not `this isoWeek` if that was your concern))

Reply:
No we're talking about different things, I was unclear, my comment was in the wrong place, and also my web browser is getting slower and slower as the number of comments on this PR grows!

I feel we really should be reviewing this by screen-sharing and discussing it - typing the comments on a PR with this much history, where the earlier comments are (rightly) resolved and so hidden, is too inefficient and taking too much time...